models/repo.go

```diff
@@ -479,6 +488,41 @@ func (repo *Repository) mustOwner(e Engine) *User {
 	return repo.Owner
 }
 
+func (repo *Repository) getOwnerName(e Engine) error {
+	if repo.OwnerName != "" {
+		return nil
+	}
+
+	if repo.Owner != nil {
+		repo.OwnerName = repo.Owner.Name
+		return nil
+	}
+
+	u := new(User)
+	has, err := e.ID(repo.OwnerID).Cols("name").Get(u)
```

Comment:
Other than a slight increase in memory usage, I don't see any point in only loading the name. The database still has to find and fetch the entire row :thinking: I would change this to just `mustOwner()`

Reply:
That's needed because it is used in migrations and can be that struct has more fields than database has so there will be error in select that database does not contain such column